Dockerfile

```diff
@@ -60,7 +60,7 @@ RUN pip3 install --user bsddb3==6.2.7
 WORKDIR /sio2/oioioi
 
 COPY --chown=oioioi:oioioi setup.py requirements.txt ./
-RUN pip3 install -r requirements.txt --user
+RUN pip3 install -r requirements.txt --user filetracker[server]
```

Comment:
It would be better to specify the version range for `filetracker`.

Reply:
But doesn't pip satisfy tbose constraints already?
In requirements.txt there is `-e .`, which afaik include setup.py constraints, which contain the version range.
And it's better to not have to update in two places.

Reply:
Yeah, you’re right.